streams/iq.m3u

```diff
@@ -1,4 +1,6 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="4kurdTV.iq",12 4kurd TV (1080p)
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="4KurdHD.fr",12 4kurd TV (1080p)
```

https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#stream-description-scheme

Reply:
Please, it's an Iraqi Kurdish music channel, not a French channel. I think you're confused.

Reply:
Channels are sorted by the country specified in the [iptv-org/database](https://github.com/iptv-org/database).

Reply:
> Channels are sorted by the country specified in the [iptv-org/database](https://github.com/iptv-org/database).

Yes I know that and I think there is an error sorting this channel.

Reply:
Nope. 4Kurd is a channel in destination of the Iraqi people indeed, but it is from a French group named Persiana. Thus, this channel is based in France, not in Iraq.

Reply:
Most of the Iraqi channels establish companies and offices in Europe, America and some countries neighboring Iraq to avoid being closed by the authorities or stormed by protesters who oppose their approach and policies.

Therefore, when you read that an Iraqi channel is based in America, this does not mean that it is an American channel, for example Al-Hurra channel.

The same thing here in 4Kurd, it is a Kurdish channel, but because its founder is from the Iranian opposition, they are forced to broadcast from France.

For this reason, it is wrong to categorize the channel according to the country in which its office or company is located, because the channel does not necessarily speak to the country in which it is located.

Reply:
While I understand your point of view, we can not do such an exception while the rules are made like this, since this channel *isn't streamed from Iraq*, but *aimed at Iraqis*.

The structure of our playlists are made as such : all the channels that you may see on our playlists are sorted in such way that defines from where they are streamed.

While it is "forced to broadcast from France" and aimed at a different group of people and/or diaspora than expected, it remains a channel that is based and streamed in France. Thus, it is a "French channel made for Iraqis".

We have a few channels that fall under that same dilemma, and we've decided to sort them under the country in which they are based at.

Reply:
Well let's end this discussion
Put the channel in the French playlist, and let the French enjoy the beautiful Kurdish music.
But you should know that this decision is unfair to the channel and its followers.

Reply:
Just to clarify, it doesn't matter which file you put the links in. 
The only thing that affects which playlist a given channel will be in is the `broadcast_area` specified in the [iptv-org/database](https://github.com/iptv-org/database). In the case of 4kurd TV it is Iran: https://iptv-org.github.io/channel?id=4KurdHD.fr If you think this information is incorrect you can leave a request to change it [here](https://github.com/iptv-org/database/issues/new?assignees=&labels=channels%3Aedit&projects=&template=channels_edit.yml&title=Edit%3A+) and if it is approved, the channel will automatically be moved to another playlist. 
And to make it all work we ask you to specify the correct `tvg-id`.

Reply:
The Kurds do not have a homeland or an official state until we classify them in it. They live in Iraq, Iran, Turkey and Syria, and they speak the same language. The only official thing they have is the Iraqi Kurdistan region, in which they enjoy a self-governing system independent of the Iraqi state.

For this reason, we always put Kurdish channels in the Iraqi playlist, because Iraq has the only official, internationally recognized gathering of Kurds.

But if we were to classify the 4kurd channel according to the nationality of the person who owns it, this means that we would classify it as an Iranian channel because the owner of the channel is an Iranian named Majid Yousefy, while the channel does not speak the Persian language, but rather the Kurdish language.

As for [submitting a request to change the channel information](https://github.com/iptv-org/database/issues/new?assignees=&labels=channels%3Aedit&projects=&template=channels_edit.yml&title=Edit%3A+), believe me, I no longer care about the matter. I will delete the channel from the Iraqi playlist, and I will leave in peace.